external/fv3fit/fv3fit/_shared/models.py

```diff
@@ -2,11 +2,97 @@
 import fsspec
 import yaml
 import os
+import numpy as np
 import xarray as xr
+import vcm
+
 from . import io
 from .predictor import Predictor
 
 
+@io.register("derived_model")
+class DerivedModel(Predictor):
+    _CONFIG_FILENAME = "derived_model.yaml"
+    _BASE_MODEL_DIRECTORY = "base_model_data"
```

Comment:
This attribute isn't used right now, you may want to delete it and add it when you implement `dump`.